models/Invite.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace humhub\modules\rest\models;
+
+use humhub\modules\space\models\Space;
+use humhub\modules\user\models\User;
+
+/**
+ * @property User|null $originator
+ * @property User|null $createdBy
+ * @property User|null $updatedBy
+ * @property Space|null $space
+ */
+class Invite extends \humhub\modules\user\models\Invite
```

Comment:
Can you please create a PR for the Core `develop` branch here.
And mark the file here that it is required up to version 1.17?

Reply:
@luke- though we needed it asap, that's why did it here instead of core. Ok, got it

Reply:
@gevorgmansuryan Your temporary solution is fine. But let's also add this in core.